.github/workflows/automerge.yml

```diff
@@ -6,5 +6,5 @@ on:
 
 jobs:
   automerge:
-    uses: oclif/github-workflows/.github/workflows/automerge.yml@main
+    uses: salesforcecli/github-workflows/.github/workflows/automerge.yml@main
```

Comment:
this won't work (secrets:inherit won't pass across github enterprise boundaries).

This would work (passing secrets explicitly)

```yml
secrets:
  SVC_CLI_BOT_GITHUB_TOKEN: {{secrets.SVC_CLI_BOT_GITHUB_TOKEN}
```